pkg/api/routes.go

```diff
@@ -60,6 +60,12 @@ func (h *Handler) GetRoutes() []Route {
 			h.CreateImage,
 		},
 		{
+			"DeleteImage",
+			"POST",
```

Comment:
@jbw976 I have added https://github.com/rook/rook/issues/500. We can decide whether or not to pull it in for 0.4